src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -19,11 +19,12 @@ public Movie add(Movie movie) {
 
     @Override
     public Movie get(Long id) {
-        return movieDao.get(id).get();
+        return movieDao.get(id).orElseThrow(() ->
+                new RuntimeException("Movie with id " + id + " not found"));
```

Comment:
Consider using `DataProcessingException` instead of `RuntimeException` for consistency with other methods and to provide more informative error handling.